################################################################################
 InvalidValueChecker_500

 Hash: InvalidValueChecker_500_89f6298522fc4795f4bd048c84f48a939ac00c02

 To attempt to reproduce this bug using restler, run restler with the command
 line option of --replay_log <path_to_this_log>.
 If an authentication token is required, you must also specify the token_refresh_cmd.

 This log may contain specific values for IDs or names that were generated
 during fuzzing, using the fuzzing dictionary. Such names will be re-played
 without modification. You must update the replay log manually with any changes
 required to execute the requests in your environment (for example, replacing
 pre-created account, subscription, or other resource IDs, as needed).
################################################################################

-> GET /api/flight/ HTTP/1.1\r\nAccept: application/json\r\nHost: localhost:8080\r\n\r\n
! producer_timing_delay 0
! max_async_wait_time 20
PREVIOUS RESPONSE: 'HTTP/1.1 500 \r\nContent-Type: application/json;charset=UTF-8\r\nTransfer-Encoding: chunked\r\nDate: Thu, 09 Jan 2025 12:44:46 GMT\r\nConnection: close\r\n\r\n1c3\r\n{"timestamp":"2025-01-09T12:44:46.319+0000","status":500,"error":"Internal Server Error","message":"Could not write JSON: Unable to find com.app.entity.Route with id 1; nested exception is com.fasterxml.jackson.databind.JsonMappingException: Unable to find com.app.entity.Route with id 1 (through reference chain: java.util.ArrayList[0]->com.app.entity.Flight[\\"route\\"]->com.app.entity.Route$HibernateProxy$kL5ROU9i[\\"name\\"])","path":"/api/flight/"}\r\n0\r\n\r\n'

